pkg/dag/graph.go

```diff
@@ -29,170 +24,45 @@ func newGraph() graph.Graph[string, string] {
 // The input is any fs.FS filesystem implementation. Given a directory path, you can call NewGraph like this:
 //
 // pkg.NewGraph(os.DirFS('path/to/directory'))
-func NewGraph(dirFS fs.FS, dirPath string) (*Graph, error) { //nolint:gocyclo
-	g := newGraph()
-
-	var packages []string
-	configs := make(map[string]build.Configuration)
-
-	err := fs.WalkDir(dirFS, ".", func(path string, d fs.DirEntry, err error) error {
-		if err != nil {
-			return err
-		}
+func NewGraph(pkgs *Packages) (*Graph, error) {
```

Comment:
Let's update the doc comment for this function to keep it accurate

Reply:
Updated